File: 백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.py
import sys


A, B, V = map(int, sys.stdin.readline().rstrip().split())

day = 0
distance = 0

V -= B


result = V // (A-B)
if V % (A -B) == 0:
    print(result)

else:
    print(result+ 1)

File: 백준/Bronze/2869. 달팽이는 올라가고 싶다/달팽이는 올라가고 싶다.c
#include <stdio.h>

int main() {

    int A, B, V = 0;
    scanf("%d %d %d",&A, &B, &V);

    int day = 0;
    V = V - A;
    
    day += V /(A - B);
    if ( V % (A -B) > 0){
        day++;
    }

    day++;
    printf("%d", day);
    
    return 0;
}
    